app/controllers/course/discussion/posts_controller.rb

```diff
@@ -69,7 +69,8 @@ def load_topic
 
   def send_created_notification(post)
     if current_course_user && !current_course_user.phantom?
-      post.topic.actable.notify(post)
+      topic_actable = post.topic.actable
+      topic_actable.notify(post) if topic_actable.respond_to? :notify
```

Comment:
prefer use brackets unless it's a DSL, `respond_to?(:notify)`

Reply:
Alright, will fix.